pdns/dnsdistdist/dnsdist-rust-lib/dnsdist-configuration-yaml-items-generated.cc

```diff
@@ -0,0 +1,223 @@
+// !! This file has been generated by dnsdist-settings-generator.py, do not edit by hand!!
+// START INCLUDE ./dnsdist-configuration-yaml-items-generated-pre-in.cc
+/*
+ * This file is part of PowerDNS or dnsdist.
+ * Copyright -- PowerDNS.COM B.V. and its contributors
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of version 2 of the GNU General Public License as
+ * published by the Free Software Foundation.
+ *
+ * In addition, for the avoidance of any doubt, permission is granted to
+ * link this program with OpenSSL and to (re)distribute the binaries
+ * produced as the result of such linking.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#include "dnsdist-configuration.hh"
+#include "dnsdist-configuration-yaml.hh"
+// END INCLUDE ./dnsdist-configuration-yaml-items-generated-pre-in.cc
+#if defined(HAVE_YAML_CONFIGURATION)
+#include "rust/cxx.h"
+#include "rust/lib.rs.h"
+#include "dnsdist-configuration-yaml-internal.hh"
+
+namespace dnsdist::configuration::yaml
+{
+void convertRuntimeFlatSettingsFromRust(const dnsdist::rust::settings::GlobalConfiguration& yamlConfig, dnsdist::configuration::RuntimeConfiguration& config)
+{
```

Comment:
So we want to overwrite the value only if it is not set to the default value. But is the `yamlConfig` part of the conditions needed? It feels redundant as setting a simple value to it's current value should not hurt. Same for `convertImmutableFlatSettingsFromRust`.